cmd/node-canary/main.go

```diff
@@ -173,7 +173,7 @@ func verifyMailserverBehavior(mailserverNode *enode.Node) {
 			Limit:          1,
 			Topic:          topic,
 			SymKeyID:       mailServerKeyID,
-			Timeout:        time.Duration(*timeout) * time.Second,
+			Timeout:        time.Duration(*timeout),
```

Comment:
@PombeirP i noticed that we always expect that plain seconds were provided in request to mail server. thus we convert it to time.Seconds explicitly by multiplying in shhext/api.go. so without this change my build was always failing, and i had to remove multiplication to seconds either here or in api.go. not sure why it was working before, will have to double-check